app/presenters/teams/filtered_list.rb

```diff
@@ -43,5 +48,9 @@ def filter_by_query
     def filter_by_id
       [Team.find(team_id)]
     end
+
+    def filter_by_favourite
```

Comment:
Is this going to be returned as array, if only one favourite selected?

Reply:
It does. Since favourite_teams is a `has_many` relation, it will always return an array